modules/spring-config/src/main/java/org/mule/runtime/config/spring/dsl/api/xml/NameUtils.java

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.config.spring.dsl.api.xml;
+
+import org.mule.runtime.extension.api.introspection.ExtensionModel;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.lang.StringUtils;
+
+/**
+ * Utilities for manipulating names of {@link ExtensionModel extensions} and
```

Comment:
this jdoc is wrong. This is a general language thing, not something related to the extension model